README.md

```diff
@@ -1,68 +1,95 @@
-
-If you are new to GitHub and run into any errors, please contact us via email at support@datajoint.com. If you are experienced with GitHub, please create an issue on the upstream repository or if you'd like help contribute, feel free to create a pull request. Please include a thorough explanantion of the error and/or proposed solution.
+    
+
+## Quick start
+
+### User installation 
+DataJoint tutorials are easily accessible using an **interactive environment** that contains all the software required to run the experiments. The setting is configured by [DevContainer] (https://containers.dev/). Here are two options to launch the interactive environment. 
+
+*Please note that to use the DataJoint Python package with an interactive environment, you need a [GitHub](https://github.com/) account.*
+
+- **Cloud-based environment: GitHub Codespaces**: (*recommended*) 
+   - This is the easiest option for **tutorial users**. You will immediately start coding using DataJoint and Python, without installing software or local environments. Cloud-based environments (IDEs), such as [GitHub Codespaces](https://github.com/features/codespaces), use built-in tools directly connected to the cloud and work on the browser.
+   
+   - Instructions:
+      - Fork the [datajoint-tutorials](https://github.com/datajoint/datajoint-tutorials) repository into your repository.
+      - From your `datajoint-tutorials` repository, click on `Code`, then click on `Codespaces` tab, and `+` option will `Create codespace on main` on your fork with default options. For more control, see the `...` where you may create `New with options...`.
+      - The building time for Codespaces is **~5m**. This is done infrequently and cached for convenience.
+      - The start time for Codespaces is **~30s**. This will pull the built codespace from the cache when you need it.
+      - *Tip*: Each month, GitHub renews a [free-tier](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#monthly-included-storage-and-core-hours-for-personal-accounts) quota of computing and storage. Typically we run into the storage limits before anything else since Codespaces consume storage while stopped. It is best to delete Codespaces when not actively in use and recreate them when needed. We'll soon be creating prebuilds to avoid larger build times. Once any portion of your quota is reached, you will need to wait for it to be reset at the end of your cycle or add billing info to your GitHub account to handle overages.
+      - *Tip*: GitHub auto names the Codespaces, but you can rename the Codespaces so that it is easier to identify later.
```

Comment:
```suggestion
      - *Tip*: GitHub auto names the Codespaces, but you can rename the Codespaces so that it is easier to identify later.
      - *Tip*: All the edits you make in these tutorial notebooks are ***not persistent***.  Edit will be reset to the original content every time you restart the server.  However, you can easily download the notebooks that interest you in keeping the changes.
```